docs/tikz/arch-multi-standby.tex

```diff
@@ -30,24 +18,7 @@
 
 \begin{tikzpicture}[>=stealth',auto,rounded corners]
 
-  \tikzstyle{app}=[circle,thick,
-	  text=aptxt,draw=apbox,fill=white,
-	  line width=0.25em,minimum size=4cm]
-
-  \tikzstyle{node}=[rectangle,minimum height=2.5cm,minimum width=4cm]
-
-  \tikzstyle{mpnode}=[rectangle split,rectangle split parts=3,
-	  align=center,
-	  rectangle split part align={center, center, left},
-	  minimum height=2.5cm,minimum width=4cm,inner sep=0.5cm]
-
-  \tikzstyle{primary}=[mpnode,text=ptxt,draw=white,
-  	  rectangle split part fill={pbox,pbox,white}]
-
-  \tikzstyle{standby}=[mpnode,text=stxt,draw=white,
-	  rectangle split part fill={sbox,sbox,white}]
-
-  \tikzstyle{monitor}=[node,text=mtxt,draw=mbox,fill=mbox]
+  \input{common.tex}
```

Comment:
This now has these empty blocks, because the definition of primary and standby changed because of the shared code.

![afbeelding](https://user-images.githubusercontent.com/1162278/95574572-70f76680-0a2d-11eb-8d64-517a917ece35.png)

Reply:
Ah yeah, that's why the arrow is shorter. Fixed now!